.travis.yml

```diff
@@ -25,23 +25,35 @@ cache:
     - $(npm config get prefix)/lib/node_modules
 
 env:
-  - GO_BUILD_TAGS=mockvpp
+  global:
+    - GO_BUILD_TAGS=mockvpp
 
 before_install:
-  - make travis
-  - make get-linkcheck
-  - make get-linters
-  - make get-dep
   - go get github.com/mattn/goveralls
 
-script:
-  - make yamllint
-  - make check-links || true
-  - make lint
-  - make dep-check
-  - make verify-binapi
-  - make
-  - make test-cover
+#script:
```

Comment:
You may want to just remove these rather than leave them commented out.

Reply:
The PR is WIP, see the label.